recipes/trouve/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "trouve" %}
+{% set version = "0.5.1" %}
+{% set author = "rwhitt2049" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_rev: {{ version }}
+  git_url: https://github.com/rwhitt2049/{{ name|lower }}
+
+build:
```

Comment:
You might be able to make this a `noarch: python` recipe:

http://conda-forge.readthedocs.io/en/latest/meta.html#building-noarch-packages